src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -241,23 +242,24 @@ private static String getValue(final String arg) {
   }
 
   public static void setupLookAndFeel() {
-    try {
-      UIManager.setLookAndFeel(getDefaultLookAndFeel());
-      // FYI if you are getting a null pointer exception in Substance, like this:
-      // org.pushingpixels.substance.internal.utils.SubstanceColorUtilities
-      // .getDefaultBackgroundColor(SubstanceColorUtilities.java:758)
-      // Then it is because you included the swingx substance library without including swingx.
-      // You can solve by including both swingx libraries or removing both,
-      // or by setting the look and feel twice in a row.
-    } catch (final Throwable t) {
-      System.out.println("LOOK AND FEEL exception: " + t.getMessage());
-      if (!GameRunner.isMac()) {
-        try {
-          UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
-        } catch (final Exception e) {
+    SwingAction.invokeAndWait(() -> {
```

Comment:
I took another look at this, looking at the source code, the indentation is actually not that bad.
The biggest problem is the control flow via exception handling, then the completely ignored exception at the end.

Simply moving stuff around to reduce the indentation is not a good fix, since there is very little here in this method. What more needs to be done is to enforce that the substance lib is actually included, and to remove the Mac special casing, and last, add a verification check to see which default look and feel shoudl be used, rather than blindly trying one, failing silently, then trying another and silently failing again.